rwengine/src/objects/CharacterObject.cpp

```diff
@@ -100,11 +100,11 @@ void CharacterObject::createActor(const glm::vec2& size)
 		physObject->setCollisionShape(physShape);
 		physObject->setCollisionFlags(btCollisionObject::CF_KINEMATIC_OBJECT);
 		physCharacter = new btKinematicCharacterController(physObject, physShape, 0.30f, 2);
-		physCharacter->setFallSpeed(20.f);
+		physCharacter->setFallSpeed(120.f);
```

Comment:
Magic numbers like this should be raised to constants and defined in an appropriate place (such as the top of the file).

I've updated the [coding conventions](https://github.com/rwengine/openrw/wiki/Coding-Conventions) for naming constants.